tests/conftest.py

```diff
@@ -19,6 +19,82 @@ def _test_environment(monkeypatch):
     monkeypatch.setenv("AWS_SESSION_TOKEN", "testing")
 
 
+@pytest.fixture
```

Comment:
That is a very good point and it also is a stronger test as it would confirm the `s3_path` parameter is parsed correctly to retrieve files from the correct location.